test/WebSites/FSharpWebSite/Controllers/HomeController.fs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace FSharpWebSite.Controllers
+
+open System
+open System.Collections.Generic
+open System.Linq
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Mvc
+
+type HomeController () =
+    inherit Controller()
+
+    member this.Index () =
+        this.View()
```

Comment:
@panesofglass Thanks - will add this once the F# support in the Common repo is built and becomes available in the MVC repo.